README.md

```diff
@@ -1,5 +1,77 @@
+
+###  Why do we need TSDoc?
+
+This scenario originated from a several tools that were all trying to process source code for projects inside Microsoft:
+
+- [Visual Studio Code](https://code.visualstudio.com): provides syntax highlighting and interactive refactoring for TypeScript doc comments
+- [TypeDoc](https://github.com/TypeStrong/typedoc): a tool that can generate an API reference web site based on doc comments
+- [DocFX](https://dotnet.github.io/docfx/):  A integrated pipeline that extracts doc comments from many different languages, but then applies its own Markdown renderer and custom tag parsing
+- [API Extractor](https://aka.ms/extractor): A tool which is used for tracking API review workflows and trimming *.d.ts files
+
+These are just examples.  For example, the **SimplrJS** authors recently joined the discussion since they use a documentation tool [ts-docs-gen](https://github.com/SimplrJS/ts-docs-gen) that wants to interoperate.  Each of these tools accepts a syntax that is loosely based on [JSDoc](http://usejsdoc.org), but encounters frustrating incompatibilities when attempting to coexist with other tools.
+
+Why can't JSDoc be the standard?  Unfortunately the JSDoc syntax is not rigorously specified, but vaguely determined by the behavior of a particular tool.  More importantly, the majority of JSDoc tags are preoccupied with providing type annotations for plain JavaScript, which is not useful for a strongly-typed language such as TypeScript.  **TSDoc** addresses these limitations, while also tackling a more sophisticated set of requirements (see below).
+
+### What are the goals?
+
+The TSDoc specification has these requirements:
+
+- **Designed for TypeScript**: ...while aligning as closely as possible with the familiar JSDoc notations we know and love.
+- **Markdown support**: Doc comments can incorporate [CommonMark](http://commonmark.org) notations for rich text elements such as boldface, code fences, headings, tables, etc.  Special accommodations are made for entrenched pipelines (e.g. GitHub Pages) that expect to bring their own Markdown renderer to the party.
+- **Common core**: Common  tags such as `@param` and `@returns` will have consistent behavior across all tools.
+- **Extensible**: Each tool can supplement the core tags with custom notations for specialized scenarios.
+- **Interoperable**: The TSDoc syntax guarantees that unsupported custom tags don't interfere with parsing of other content. (For example, if you didn't recognize the `@remarks` or `@link` tag in our example above, what to do with the text between or after? Are they part of the tag or not?) TSDoc also avoids Markdown ambiguities.  (How would you handle a single backtick inside a `{@link}` tag? I can't even write this example because my Markdown editor chokes on it!) 
```

Comment:
>(How would you handle a single backtick inside a `{@link}` tag? I can't even write this example because my Markdown editor chokes on it!)  [](start = 333, length = 138)

?